xrayvision/visibility.py

```diff
@@ -39,8 +152,8 @@ class Visibility:
         The x, y offset of phase center
 
     """
-    @apu.quantity_input(uv=1/apu.arcsec, offset=apu.arcsec, center=apu.arcsec, pixel_size=apu.arcsec)
-    def __init__(self, vis, *, u, v, offset=(0., 0.) * apu.arcsec, center=(0., 0.) * apu.arcsec):
+    @apu.quantity_input(u=1/apu.arcsec, v=1/apu.arcsec)
+    def __init__(self, visibilities, *, u, v, names, uncertainty=None, meta=None):
```

Comment:
I think this is what we need to make sure all the inputs are the correct types.  We should confirm this is correct though.

```suggestion
    @apu.quantity_input()
    def __init__(self,
                 visibilities: apu.Quantity,
                 *,
                 u: apu.Quantity[1/apu.arcsec],
                 v: apu.Quantity[1/apu.arcsec],
                 names: Iterable[str],
                 uncertainty: Any = None,
                 meta=None):
```